pages/overview.md

```diff
@@ -1,13 +1,44 @@
+
+Besides the goal to develop an anomaly detection algorithm to distinguish between hybrids and non-hybrids for one species, this challenge aims to investigate whether such an algorithm is generalizable to the other visually mimicking species. 
+
+![subspecies_to_hybrids comparison different subspecies of Species A compared to two of its mimics](https://github.com/user-attachments/assets/8647e1f5-4f99-48c6-8325-fdfd0e5d4c21)
+
+
+## Setup Overview
+This challenge is designed to simulate a real-world biological scenario. Suppose a biologist studies a particular butterfly Species A with many subspecies. One day, the biologist finds that a subset of the images collected looks slightly abnormal in their visual appearance. The biologist does not recognize the pattern as belonging to any of the subspecies on which their research is focused. After investigation, the biologist finds that these unusual samples are hybrids produced by different subspecies of Species A. Realizing that they may encounter other hybrids in future collections of images, the biologist seeks an anomaly detection algorithm to automatically identify (unseen) hybrid cases.
+
+### Algorithm requirement
+The developed anomaly detection algorithm needs to output an anomaly score (a real number) for each test image. The higher the score is, the more likely the image is an anomaly (i.e., hybrid).
+
+### Training data
+The training data comprises images from all the Species A subspecies and the most common hybrid. The most common* hybrid refers to a specific combination of the parent subspecies that has the most images. This hybrid is called the signal hybrid; other hybrids are called the non-signal hybrids.    
+
+### Test data
+We consider two sets of images in the test set:
+- One from Species A, comprising images from all the Species A subspecies, the signal hybrid, and the non-signal hybrids.
+- One from a mimicking Species B, comprising images from two subspecies of Species B and their hybrid. The two subspecies of Species B are the ones mimicking the parent subspecies of the signal hybrid of Species A.
 
 ## Timeline
 
-This ML Challenge starts on June 15, 2024 and will run through September [end day], 2024. Be sure to resubmit your preferred model from the development phase by [date, time]; it will then be run on the test sets to detect hybrids of Species A and its major hybrid mimics. Only one submission will be run against the test sets to determine your final score in the challenge.
-  
+This ML Challenge starts on September [start day], 2024, and will run through [end date]. Be sure to resubmit your preferred model from the development phase by [date, time]; it will then be run on the final test sets. Only one submission will be run against the test sets to determine your final score in the challenge.
```

Comment:
```suggestion
This ML Challenge starts on September 11th, 2024, and will run through January 17th, 2025. Be sure to resubmit your preferred model from the development phase by January 17th at 11:59pm [AOE](https://www.timeanddate.com/time/zones/aoe); it will then be run on the final test sets. Only one submission will be run against the test sets to determine your final score in the challenge.
```
Update challenge deadlines.